sql/ddl_api.sql

```diff
@@ -57,8 +59,9 @@ CREATE OR REPLACE FUNCTION @extschema@.create_distributed_hypertable(
     chunk_target_size       TEXT = NULL,
     chunk_sizing_func       REGPROC = '_timescaledb_internal.calculate_chunk_interval'::regproc,
     time_partitioning_func  REGPROC = NULL,
-    replication_factor      INTEGER = 1,
-    data_nodes              NAME[] = NULL
+    replication_factor      INTEGER = NULL,
+    data_nodes              NAME[] = NULL,
+    distributed             BOOLEAN = true
```

Comment:
I think we should elide this argument for `create_distributed_hypertable`. It doesn't make sense to have it here IMO. It is always "true" and cannot be changed.

```suggestion
   
```

Reply:
I can do it , but I was thinking to just make this function an exact alias to the `create_hypertable()`, otherwise internally we will have to rely on the `dist_call` argument to see if the function was actually been called when we check for the call arguments. Also there would be gaps in argument enumeration, when a new argument will be added after the `distributed` in the future

Reply:
I understand the desire to make the signatures the same, but IMO, it doesn't make sense to have this particular parameter for the "distributed" function. So, I don't think we should add parameters that don't make sense just to keep the signatures the same.